Workflow/Sources/AnyWorkflowConvertible.swift

```diff
@@ -147,3 +147,18 @@ struct OutputBlockingWorkflow<Child: AnyWorkflowConvertible>: Workflow {
             .rendered(in: context)
     }
 }
+
+// MARK: -
+
+extension AnyWorkflowConvertible where Output == Never {
+    /// A convenience for workflows that don't produce output but want to change the declared output type.
+    ///
+    /// This equivalent to `.mapOutput { switch $0 {} }` but avoids the compiler warning for unreachable code in the provided transform.
+    public func mapOutput<NewOutput>() -> AnyWorkflow<Rendering, NewOutput> {
+        asAnyWorkflow().mapOutput(silenceUnreachableWarning(_:))
+    }
+
+    private func silenceUnreachableWarning<T>(
```

Comment:
nit: let's move this out of the extension so it's not implicitly nested in a generic type, and rename it for the substance of what it's doing. also it seems the compiler allows you to implement this without a method body, so it can be reduced to something like:

```swift
private func mapNever<T>(_ input: Never) -> T {}
```

Reply:
Good call, much cleaner